<ui version="4.0" >
 <class>config::AddSensorDialog</class>
 <widget class="QDialog" name="config::AddSensorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Sensor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QVBoxLayout" name="addSensorLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="sensorFieldsLayout" >
       <item>
        <widget class="SensorCatalogWidget" name="SensorCatTbl" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="textFieldsLayout" >
         <property name="sizeConstraint" >
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <spacer name="verticalSpacer_top" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="SensorLayout" >
           <item>
            <widget class="QLabel" name="SensorLabel" >
             <property name="text" >
              <string>Sensor Name</string>
             </property>
             <property name="buddy" >
              <cstring>SensorText</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SensorText" >
             <property name="inputMask" >
              <string/>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="DeviceLayout" >
           <item>
            <widget class="QLabel" name="DeviceLabel" >
             <property name="text" >
              <string>Device</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DeviceText" >
             <property name="inputMask" >
              <string/>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="frame" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="IdLayout" >
           <item>
            <widget class="QLabel" name="IdLabel" >
             <property name="text" >
              <string>Sensor Id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="IdText" >
             <property name="inputMask" >
              <string/>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_bottom" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SensorCatalogWidget</class>
   <extends>QTableWidget</extends>
   <header>SensorCatalogWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config::AddSensorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config::AddSensorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SensorCatTbl</sender>
   <signal>currentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</signal>
   <receiver>config::AddSensorDialog</receiver>
   <slot>setSensorText(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>99</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
